tests/integration/data_processing/test_data_processor.py

```diff
@@ -401,3 +401,133 @@ def test_update_transformers_id_generator(self):
         )
         with pytest.raises(SynthesizerInputError, match=error_msg):
             dp.update_transformers({'user_id': UniformEncoder()})
+
+    def test_numerical_dtype_handling(self):
```

Comment:
Do you think `pytest.mark.parametrize` is better to use for these? You could make the dtype the input

Reply:
Yes this is good advice, thanks! Done in [5f4a918](https://github.com/sdv-dev/SDV/pull/2184/commits/5f4a918cc4a1fd8b76b21756572b5864cded84af)